### ISSUE TITLE
[haxcms] Cannot read property 'setAttribute' of undefined     at HTMLElement._handleKey (a11y-tabs.js:770)

### ISSUE THREAD

**Issue by btopro** [elmsln/issues#682]
something w/ a11y tabs and run time registration of the form w/ them in it

**Comment by btopro** [comment]
fixed, better UX on tabs now too if window scrolls / has other listeners
